tasks/FindMyMates.tex

```diff
@@ -0,0 +1,52 @@
+\section{Find My Mates [Party Host]}
+
+The robot starts in a room and waits for someone to approach it. The guest knows the name of other guests at the party but doesn’t know how these people look like and where the people are. The robot will gather this information for the guest.
+
+
+\subsection{Main Goal}
+The robot provides the location and unique description of 3 guests of the party as required by another guest.
+
+\noindent\textbf{Reward:} 750 points (250 points per guest).
+
+\subsection{Bonus rewards}
+\begin{enumerate}[nosep]
+	\item Robot providing unique description of a person without having asked such information (75 points)
+\end{enumerate}
+
+
+\subsection{Setup}
+\begin{itemize}
+	\item \textbf{Location}: The robot is in a predefined room at least one person that will approach it.
+
+	\item \textbf{People}: 5 people distributed around the house.
+\end{itemize}
+
+\subsection{Additional rules and remarks}
+\begin{enumerate}[nosep]
+	\item \textbf{Deus ex Machina:} Score reduction for requesting human assistance is applied per guest.
+
+	\item \textbf{Calling the operator for the final answer:} 50\% reduction if the robot does not go back autonomously to the operator to report the information.
+	
+	\item \textbf{Penalty:} If the robot talks to an empty room or does not clearly face a person or crowd when talking (e.g. speaks to a wall), -75 points of penalty.
+\end{enumerate}
+
+
+\subsection{Referee instructions}
+
+The referees need to
+\begin{itemize}
+	\item Make sure there every person has a name.
+	\item Make sure people stay roughly in the same place.
+	\item Distribute the people in the rooms.
+\end{itemize}
+
+\subsection{OC instructions}
+
+2h before test:
+\begin{itemize}
+	\item Announce the location where the robot will be instructed.
```

Comment:
Missing:
- Assign predefined names to the guests
- Announce predefined room in which test takes place

Reply:
Should also have,

* Assign predefined location to each person.

If we want the robot to report the location too.

Also, OC should announce the list of names. This way the robot will know the list of names before the test starts which will save time to achieve the main goal.

Reply:
That's definitively a No-Go from all perspectives. Wherever the robot is looking for people, the people shall not have assigned a predefined location. Even less two hours before the test. We don't know which volunteers we can rely on and for how long. Further, people must be swapped to prevent cheating.

Regarding the list of names, that's published during setup days. It's in General Rules